components/slider/script.js

```diff
@@ -228,30 +228,30 @@ class Slider extends BaseComponent {
      * @param {HTMLElement} scrollableContainer
     */
     resize(scrollableContainer) {
-        components.waitForFrames(() => {
-            const scrollableContainerSize = this.getElementSize(scrollableContainer);
-            const sliderWrapper = this.querySelector(`.guic-${this.orientation}-slider-wrapper`);
-
+        const sliderWrapper = this.querySelector(`.guic-${this.orientation}-slider-wrapper`);
+        const scrollableContainerComponent = scrollableContainer?.parentElement?.parentElement;
+        const scrollableContainerSize = this.getElementSize(scrollableContainer);
+        if (scrollableContainerComponent.tagName === 'GAMEFACE-SCROLLABLE-CONTAINER' && !scrollableContainerComponent.hasAttribute('fixed-slider-height')) {
```

Comment:
Can the tagname be different?

Reply:
No. We relay that the wrapper should be `gameface-scrollable-container`.